satpy/composites/__init__.py

```diff
@@ -41,6 +41,11 @@
                               'isneginf', 'isposinf']
 
 
+def merge_tuples(*t):
+    """Merge tuples and tuples of tuples into one tuple."""
+    return tuple(j for i in (t) for j in (i if isinstance(i, tuple) else (i,)))
```

Comment:
Could you please use longer names for `i`, `j`, and `t`? This is very hard to read. Why the extra parentheses around `(t)`? When is `i` not a tuple?

Reply:
You are right. This is quite bad :-D. I will make it more readable.